metadoc-js/src/main/scala/metadoc/ScalaDocumentSymbolProvider.scala

```diff
@@ -42,22 +42,26 @@ class ScalaDocumentSymbolProvider(index: Index)
   }.toMonacoThenable
 
   def symbolKind(denotation: Denotation): Option[SymbolKind] = {
-    if (denotation.isPARAM || denotation.isTypeParam)
+    import denotation._
+
+    if (isPARAM || isTypeParam)
       None
-    else if (denotation.isVal || denotation.isVar)
+    else if (isVal || isVar)
       Some(SymbolKind.Variable)
-    else if (denotation.isDef)
+    else if (isDef)
       Some(SymbolKind.Function)
-    else if (denotation.isPrimaryCtor || denotation.isSecondaryCtor)
+    else if (isPrimaryCtor || isSecondaryCtor)
       Some(SymbolKind.Constructor)
-    else if (denotation.isClass)
+    else if (isClass)
       Some(SymbolKind.Class)
-    else if (denotation.isObject)
+    else if (isObject)
       Some(SymbolKind.Object)
-    else if (denotation.isTrait)
+    else if (isTrait)
       Some(SymbolKind.Interface)
-    else if (denotation.isPackage || denotation.isPackageObject)
+    else if (isPackage || isPackageObject)
       Some(SymbolKind.Package)
+    else if (isType)
+      Some(SymbolKind.Namespace) // Note: no type related symbol kind exists
```

Comment:
vscode seems to use the same symbol kind for constructors and enum values as it does for type aliases 
<img width="268" alt="screen shot 2017-07-04 at 12 31 56" src="https://user-images.githubusercontent.com/1408093/27826798-ca5562f2-60b4-11e7-89a6-082c8ed35908.png">

<img width="306" alt="screen shot 2017-07-04 at 12 30 39" src="https://user-images.githubusercontent.com/1408093/27826785-b413f030-60b4-11e7-80bb-8a248e838ebb.png">